modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -268,9 +313,21 @@ private DefaultCouchbaseEnvironment createCouchbaseEnvironment() {
         initCluster();
         return DefaultCouchbaseEnvironment.builder()
             .bootstrapCarrierDirectPort(getMappedPort(11210))
-            .bootstrapCarrierSslPort(getMappedPort(11207))
             .bootstrapHttpDirectPort(getMappedPort(8091))
-            .bootstrapHttpSslPort(getMappedPort(18091))
             .build();
     }
+
+    @Getter
+    @RequiredArgsConstructor
+    protected enum Ports {
+        CAPI("CAPI_PORT", 8092),
```

Comment:
I think this list needs to be extended):

kv: 11210
kv ssl: 11207
mgmt: 8091
mgmt ssl: 18091
capi: 8092
capi ssl: 18092
n1ql: 8093
n1ql ssl: 18093
fts: 8094
fts ssl: 18094
analytics: 8095
analytics ssl: 18095

Reply:
I think not all of them (especially SSL ones) can be changed here:
https://github.com/couchbase/docker/blob/e295b6e494555dab0259e067ab535a5c01937cfd/community/couchbase-server/5.1.1/scripts/entrypoint.sh#L40

i.e. there is no fts ssl :(

Reply:
so you've been looking at 5.1.1 community - I checked 5.5 beta enterprise which should be the latest https://github.com/couchbase/docker/blob/master/enterprise/couchbase-server/5.5.0-beta/scripts/entrypoint.sh .. but even there I don't see it.. and I think this is a bug in the script.

can you include it in this PR and I'll drop the author an email and will follow up

Reply:
I'm afraid we cannot change it atm because 5.5 is still in beta and also we have to support multiple versions of CB

However, we could generate the config ourselves and put it into the container

Reply:
well, would it be possible to just supply those env variables and they'll be noops on older versions?

looking through the port mapping I think those are missing from the couchbase docker file to override:

- fts ssl (“fts_ssl_port”)
- data ssl (“memcached_ssl_port”)
- analytics
- analytics ssl